src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export const Person = ({ person:
+  { name, age, sex, isMarried, partnerName } }) => (
+
+    <section className="Person">
+      {name && (
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      )}
+      {age && (
+      <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">{isMarried ? `My ${sex === 'm' ? 'wife' : 'husband'}'s name is ${partnerName}` : 'I am not married'}</p>
```

Comment:
And better move this code `sex === 'm' ? 'wife' : 'husband'` to separate variable and use it here.
```suggestion
      <p className="Person__partner">
        {isMarried 
          ? `My ${sex === 'm' ? 'wife' : 'husband'}'s name is ${partnerName}` 
          : 'I am not married'}
      </p>
```